### DISCUSSION TITLE
Do not recommend Authy anymore as Authenticator app

### DISCUSSION THREAD

**Discussion by poisonborz** [dani-garcia/vaultwarden#3891] [Show and tell]
It may not be the best place to file this, but I feel this is a security issue so rather than starting a discussion I would firstly file this as an issue to be fixed.

Please do not recommend Authy as a TOTP app. Since they were acquired by Twilio the app sunk in quality. It's closed source. They had a security breach in 2022. It doesn't allow to export/transfer your auths. Even deleting your account is really hard.

Instead:
Aegis on Android
Raivo on iOS

Should trust your own research but these seem to be the community recommended choices currently.

**Comment by BlackDex** [comment]
I'm not sure why you place this here at all.
As far as i know we do not mention Authy or Aegis or Raivo or Google Authenticator or Microsoft Authenticator at all anywhere or recommend anything over something else.

So, creating an issue here, or even a discussion doesn't really add anything in my opinion.

**Reply by stefan0xC** [reply]
I've never heard of these apps so I don't think that it's totally without merit but yeah I would rather we have a discussion about what to look for in 2FA apps (and why) than which specific apps fulfills all requirements (especially because those tend to change over time) or which don't but I'm also not sure it's necessary to have such a discussion here. :shrug: 

Regarding Authy: I don't think I would have recommended that app even before this acquisition but that also happened ages ago (in February 2015) so not sure why you even bring it up now.

**Comment by poisonborz** [comment]
There are recommendations on the web vault 2FA setup page - if this isn't part of the maintained code of this project, then sorry for the misunderstanding.

![image](https://github.com/dani-garcia/vaultwarden/assets/1070611/33124569-f7e4-4e61-8c5a-1c9cd9218cf1)

**Reply by stefan0xC** [reply]
Oh, thanks for pointing that out. Could you maybe make a report in the https://github.com/bitwarden/clients repository? Because we don't really control the web-vault but just ship a slightly modified version. So upstream would be a better place to file this issue.
